cdbot/cogs/fun.py

```diff
@@ -238,6 +238,11 @@ def check(reaction, user):
             )
         ):
             await message.add_reaction("\N{WAVING HAND SIGN}")
+        # Simple regex responses.
+        if ".hac" in message.content.lower():
+            await message.channel.send("Charging the Low Orbit Ion Canon, please stand by!")
+        elif ".dox" in message.content.lower():
+            await message.channel.send("OK, scraping their parent's public Facebook feed!")
```

Comment:
This isn't a regex and it'll be very greedy and fire a lot of the time it shouldn't. Why can't they just be commands like everything else